scripts/ci/publishing-needed.js

```diff
@@ -0,0 +1,49 @@
+import path, { dirname } from "path";
+import fs from "fs/promises";
+import { spawnSync } from "child_process";
+import { fileURLToPath } from "url";
+
+if (process.argv.length < 3) {
+  throw new Error(`Usage: node publishing-needed.js <name-of-the-workspace>`);
+}
+const workspace = process.argv[2];
+
+(async function main() {
+  const pluginsFolderPath = path.join(
```

Comment:
maybe `resolve` instead, and then you can do `"../../workspaces", workspace, "plugins"` as the argument list, no need to keep them all separate, you'll get them out with native slashes iirc